jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -497,9 +500,15 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 	}
 
 /*[IF JAVA_SPEC_VERSION >= 20]*/
+/*[IF JAVA_SPEC_VERSION >= 22]*/
+	public void ensureNativeAccess(Module mod, Class<?> clzOwner, String methodName, Class<?> clzCaller) {
+		mod.ensureNativeAccess(clzOwner, methodName, clzCaller);
```

Comment:
Please use the same prefix (`cls`) as the existing method.

Reply:
Updated